scripts/update-bq.ts

```diff
@@ -21,6 +21,8 @@ const fieldsToKeep = [
 const tokensInfo = getTokensInfoByNetworkIds([
   NetworkId['celo-mainnet'],
   NetworkId['ethereum-mainnet'],
+  NetworkId['arbitrum-one'],
```

Comment:
In https://github.com/valora-inc/valora-rest-api/pull/883 I named these `optimism-mainnet ` and `arbitrum-mainnet`, but it's unclear if that's the best name for for these chains. There is this [doc](https://github.com/valora-inc/engineering/blob/main/edrs/0003-naming-blockchain-networks.md) which gives a guide on how to name networks; however, arbitrum and optimism are not covered.

Reply:
@silasbw what should we name these chains?

Reply:
I chatted with @silasbw earlier and he mentioned using the naming [here](https://github.com/valora-inc/wallet/blob/f5ffd2ec942bb971081d4b5398631acba0fd7c50/src/transactions/types.ts#L12), but I am flexible either way. Please let me know the best way to proceed!